proto-lens-json.cabal

```diff
@@ -23,8 +23,8 @@ library
   other-modules: Data.ProtoLens.JSON.Decode
                  Data.ProtoLens.JSON.Encode
                  Data.ProtoLens.JSON.Util
-  build-depends: base >= 4.8 && < 4.17,
-                 aeson >= 2.0 && < 2.1,
+  build-depends: base >= 4.8 && < 4.19,
+                 aeson >= 2.0 && < 2.2,
```

Comment:
Hmm, even without changing these bounds, we could use this internally. I guess Nix is ignoring the bounds for Stackage snapshots somehow?